api-reference/10 UI Components/dxCalendar/1 Configuration/showWeekNumbers.md

```diff
@@ -5,7 +5,7 @@ default: false
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether or not the UI component displays a column with the week numbers.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The [weekNumberRule](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#weekNumberRule) property defines the beginning of the first week of year.
```

Comment:
`weekNumberRule` has its own description. Let's just mention this property in the See Also section.